tests/debugging/exploration/__init__.py

```diff
@@ -0,0 +1,2 @@
+# To run an exploration test, set
+# PYTHONPATH=/path/to/tests/debugging/exploration/
```

Comment:
is it worth configuring this as a runpy module? python -m tests/debugging/exploration?

A `__main__` module could be a little more user friendly than needing to set pythonpath and `DD_DEBUGGER_EXPL_COVERAGE_ENABLED`

Reply:
All good points, but considering that this is mostly for testing and internal use, I'd be a bit reluctant to add extra complexity here